pydantic_ai_slim/pydantic_ai/messages.py

```
@@ -2960,8 +2960,82 @@ class OutputToolResultEvent(ToolResultEvent):
+    results: DeferredToolResults
+    """The resolved results for the deferred tool calls."""
+
+    _: KW_ONLY
+
+    event_kind: Literal['deferred_tool_result'] = 'deferred_tool_result'
+    """Event type identifier, used as a discriminator."""
+
+    __repr__ = _utils.dataclasses_no_defaults_repr
+
+
+@deprecated(
```

Ruff (0.15.20)

[error] Undefined name `deprecated`

(F821)